source/core/shape/ovus.cpp

```diff
@@ -45,16 +45,18 @@
 *
 *  ovus
 *  {
-*     bottom_radius,top_radius
+*     bottom_radius,top_radius [distance d] [radius r] [precision p]   [sturm]
 *  }
 *
 *  The so long awaited 'Egg' forms.
 *
 *  Normally, the bottom_radius is bigger than the top_radius
-*  the center of the top sphere is at the zenith of the bottom sphere
+*  the center of the top sphere is at the zenith of the bottom sphere, unless
+*    a greater distance d is specified with "distance"
```

Comment:
It's the distance from the origin, it must be at least the radius of the bottom sphere, but can be bigger (previously, it was always the radius of the bottom sphere)